hermes-management/build.gradle

```diff
@@ -43,22 +43,29 @@ dependencies {
     testCompile group: 'org.testcontainers', name: 'kafka', version: versions.testcontainers
 }
 
+def consoleDir = "${buildDir}/generated/console"
+
 task buildHermesConsole(type: Exec) {
-    workingDir '../hermes-console'
+    def dir = '../hermes-console'
+    workingDir dir
+    outputs.dir(dir + "/dist")
+    inputs.dir(dir + "/static")
     commandLine './package.sh'
 }
 
 task attachHermesConsole(type: Copy, dependsOn: 'buildHermesConsole') {
-    from '../hermes-console/dist/static'
-    into(sourceSets.main.output.resourcesDir.path + '/static')
-}
-
-tasks.register('prepareIndexTemplate') {
+    def srcDir = '../hermes-console/dist/static'
+    def targetDir = consoleDir + '/static'
+    from srcDir
```

Comment:
Small improvement, you should be able to replace this by `from tasks.named('buildHermesConsole')` and you can remove the `dependsOn: 'buildHermesConsole'`

Reply:
done. I also had to rework this task a little bit to have `dist` as a source (and `static` part included when copying)